compass/compass.js

```diff
@@ -0,0 +1,53 @@
+﻿API.onUpdate.connect(function (sender) {
+
+    if (0.3 < API.getGameplayCamDir().X && 0.3 < API.getGameplayCamDir().Y) {
+
+        API.drawText("NE", 350, 1000, 1, 255, 255, 255, 255, 2, 1, false, true, 0);

```

Comment:
what do you mean, even if i do that, i would still need to do drawText every single time??